networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages DNS external to the node.
+
+These release notes track the development of the ExternalDNS Operator in {product-title}.
+
+[id="external-dns-operator-release-notes-1.2.0"]
+== ExternalDNS Operator 1.2.0
+
+The following advisory is available for the ExternalDNS Operator version 1.2.0:
+
+* link:https://errata.devel.redhat.com/advisory/123003[RHEA-2022:5867 ExternalDNS Operator 1.0 operator/operand containers]
```

Comment:
We don't actually link to the errata. Customers won't be able to see this because it's behind our VPN. We link to access.redhat.com. So the correct link would be:

And would you want 1.2 here? because I think that's the actual release: 

```suggestion
* link:https://access.redhat.com/errata/RHEA-2023:7239[RHEA-2022:5867 ExternalDNS Operator 1.2 operator/operand containers]
```